packages/core-extensions/content-containers/structured-content/src/edit/index.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div class="bg-primary-darken-3">
+    <VAlert
+      v-if="!subcontainers.length"
+      class="my-8"
+      color="primary-lighten-4"
+      icon="mdi-information-outline"
+      variant="outlined"
+      prominent
+    >
+      Click the button below to add a first content section.
+    </VAlert>
+    <VRow
+      v-for="subcontainer in subcontainers"
+      :key="subcontainer.id"
+      class="subcontainer-list"
+    >
+      <VCol>
+        <StructuredSubcontainer
+          v-bind="config[subcontainer.type]"
+          :activities="activities"
+          :elements="elements"
+          :container="subcontainer"
+          :is-disabled="disabled"
+          @add:element="emit('add:element', $event)"
+          @save:element="emit('save:element', $event)"
+          @delete:element="emit('delete:element', $event)"
+          @reorder:element="emit('reorder:element', $event)"
+          @update:subcontainer="emit('update:subcontainer', $event)"
+          @delete:subcontainer="emit('delete:subcontainer', $event)"
```

Comment:
should this also be `@delete:subcontainer="(...e) => emit('delete:subcontainer', ...e)"` as in `StructuredSubcontainer`?

Reply:
Aligned 🧹